run-container.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+echo ">> Waiting for postgres to start"
+WAIT=0
+while ! nc -z "db" 5432; do
```

Comment:
You could also run something more PostgreSQL-specific like `psql postgres postgres -c "select 1"`. This way you'll ensure that PostgreSQL is actually up and not just responding to the TCP port, but that's minor.

Reply:
Good idea, I ll update that